securedrop_export/entrypoint.py

```diff
@@ -0,0 +1,23 @@
+import os
+import shutil
+import sys
+
+from securedrop_export import export
+from securedrop_export import main
+
+
+def start():
+    my_sub = export.SDExport(sys.argv[1])
+    try:
+        # Halt immediately if target file is absent
+        if not os.path.exists(my_sub.archive):
+            msg = "File does not exist"
+            my_sub.exit_gracefully(msg)
+        main.__main__(my_sub)
+        # Delete extracted achive from tempfile
+        shutil.rmtree(my_sub.tmpdir)
```

Comment:
Archive might remain here if there's an exception in the main function, and thus the files might accumulate on sd-export-usb. Not a huge deal since it's a disposable VM, but performing the removal in exit_gracefully per the previous iteration did guard against this

Reply:
removed by accident in exit_gracefully! Restoring it.

Reply:
fixed